src/spec/payid-protocol.md

```diff
@@ -116,11 +116,11 @@ This specification defines the PayID protocol - an application-layer protocol, w
 # JSON Format Design
   JSON, as described in [RFC8259][], defines a test format for serializing structured data. Objects are serialized as an unordered collection of name/value pairs. JSON does not define any semantics around the name/value pairs that make up an object. PayID protocol's JSON format defines name/value pairs that annotate a JSON object, property or array for PayID protocol resources.
 
-  The PayID client MUST request a PayID response in JSON format through the `Accept` header with the media type as defined below, optionally followed by format parameters. One of the optional parameters is the case-insensitive `q` value as described in Section 5.3.1 of [RFC7231][] to indicate relative preference.
+  The PayID client MAY request a PayID response in JSON format through the `Accept` header with the media type as defined below, optionally followed by format parameters. One of the optional parameters is the case-insensitive `q` value as described in Section 5.3.1 of [RFC7231][] to indicate relative preference.
 
   Each message body is represented as a single JSON object. This object contains a name/value pair whose value is the correct representation for a primitive value as described in [RFC8259][], or an array or object as described in the section below.
 
-  If the PayID server does not support JSON format, it MUST reply with an appropriate error response.
+  If the PayID server does not support JSON format, it SHOULD reply with an appropriate error response.
```

Comment:
Maybe I shouldn't change this one, but I thought PayID servers were required to support JSON payloads, so I don't even know how you could end up in this situation.